### PR TITLE
Add Netherlands provider

### PR DESCRIPTION
**Pull request by guushoekman**
I followed the instructions [here](https://transportr.app/contribute/) to add the Netherlands provider (https://navitia.opendatasoft.com/explore/dataset/nl/information/).

I''ll add a few comments where I was unsure about what exactly to add.

Hope this is useful, please let me know if anything needs to be fixed.

### CODE REVIEW

File: test/de/schildbach/pte/live/NetherlandsProviderLiveTest.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    }
+
+    @Test
+    public void nearbyStationsAddress2() throws Exception {
+        nearbyStationsAddress(52069885, 4320715);
+    }
+
+    @Test
+    public void nearbyStationsStation() throws Exception {
+        nearbyStationsStation("stop_point:OAM:2323791");
```
Comment (guushoekman) [inline test/de/schildbach/pte/live/NetherlandsProviderLiveTest.java:47]: I saw some comment in this existing file: https://github.com/schildbach/public-transport-enabler/blob/master/test/de/schildbach/pte/live/FranceNorthEastProviderLiveTest.java#L51

There, it says:

```
// station to find other stations around
// look in NTFS file for a stop_id (that contains "SP") and apend to "stop_point:" 
```

In the Netherlands NTFS file, file `object_codes.txt` I found this line:

`stop_point,OAM:2345074,source,2345074`

I'm not sure if I used the right formatting here.

File: test/de/schildbach/pte/live/NetherlandsProviderLiveTest.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    }
+
+    @Test
+    public void nearbyStationsInvalidStation() throws Exception {
+        nearbyStationsInvalidStation("stop_point:OAM:23237945");
+    }
+
+    @Test
+    public void queryDeparturesEquivsFalse() throws Exception {
+        queryDeparturesEquivsFalse("stop_point:OAM:2323791");
```
Comment (guushoekman) [inline test/de/schildbach/pte/live/NetherlandsProviderLiveTest.java:67]: Here I used the same as nearbyStationsStation, like in the FranceNorthEastProvider file

File: test/de/schildbach/pte/live/NetherlandsProviderLiveTest.java [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    }
+
+    @Test
+    public void queryDeparturesStopArea() throws Exception {
+        queryDeparturesStopArea("stop_area:OAM:stoparea:390116");
```
Comment (guushoekman) [inline test/de/schildbach/pte/live/NetherlandsProviderLiveTest.java:72]: From the NTFS file I used the line

`stop_area,OAM:stoparea:390116,source,stoparea:390116`

Not sure if I formatted it correctly.

**Review by guushoekman — COMMENTED** [review]
Have added some comments

**Comment by schildbach** [comment]
Thanks for your contribution!

I'm hesitant to merge this, because it is based on Navitia, and support for Navitia is in bad shape in general. We need someone to take the role of maintaining AbstractNavitiaProvider.

As a consequence, I'm about to remove all Navitia based providers. See #402.

**Comment by schildbach** [comment]
Thanks a lot for your contribution. Unfortunately, due to the current bad shape of Navitia support in public-transport enabler, time constraints on our side and due to the centralized nature of navitia.io I have decided to drop support for Navitia. Thus, I'm going to close this PR. Sorry for the inconvenience and thanks again.

**Pull request closed without merging**